inst-coffee.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="milestone1.css"
+</head>
+<body>
+    <!--Website header - should be the same throughout the pages-->
+    <header class="header">
+        <a class="title-section" href="milestone1.html">
+            <h1 class="title">Arya's Best Cafe</h1>
+        </a>
+        <nav>
+            <a class="navigator" href="milestone1.html">Home</a>
+            <a class="navigator" href="about-me.html">About Me</a>
+        </nav>
+    </header>
```

Comment:
You never used the header class in your css. If you wanted to put the title and links on the same level you could do so by using a flexbox on this header element as opposed to the navigator class